podman/domain/networks_manager.py

```diff
@@ -0,0 +1,221 @@
+                You might want this to get the containers attached to them. (ignored)
+        """
+        compatible = kwargs.get("compatible", True)
+
+        filters = kwargs.get("filters", dict())
+        filters["name"] = kwargs.get("names", None)
+        filters["id"] = kwargs.get("ids", None)
+        filters = api.format_filters(filters)
+
+        params = {"filters": filters}
+        path = f"/networks{'' if compatible else '/json'}"
+
+        response = self.client.get(path, params=params, compatible=compatible)
+        body = response.json()
+
+        if response.status_code != 200:
+            raise APIError(body["cause"], response=response, explanation=body["message"])
+
+        nets: List[Network] = list()
+        for item in body:
+            nets.append(self.prepare_model(item))
+        return nets
+
+    # pylint is flagging 'network_id' here vs. 'key' parameter in super.get()
+    def get(self, network_id: str, *_, **kwargs) -> Network:  # pylint: disable=arguments-differ
+        """Return information for network network_id.
+
+        Args:
+            network_id: Network name or id.
+
+        Keyword Args:
+            compatible (bool): Should compatible API be used. Default: True
+
+        Raises:
+            NotFound: Network does not exist.
+            APIError: Error returned by service.
+
+        Note:
+            The compatible API is used, this allows the server to provide dynamic fields.
+                id is the most important example.
+        """
+        compatible = kwargs.get("compatible", True)
+
+        path = f"/networks/{network_id}" + ("" if compatible else "/json")
+
+        response = self.client.get(path, compatible=compatible)
+        body = response.json()
+
+        if response.status_code != 200:
+            if response.status_code == 404:
+                raise NotFound(body["cause"], response=response, explanation=body["message"])
+            raise APIError(body["cause"], response=response, explanation=body["message"])
+
+        if not compatible:
+            body = body[0]
+
+        return self.prepare_model(body)
+
+    def create(self, name: str, *_, **kwargs) -> Network:
+        """Create a Network.
+
+        Args:
+            name: Name of network to be created
+
+        Keyword Args:
+            attachable (bool): Ignored, always False.
+            check_duplicate (bool): Ignored, always False.
+            disabled_dns (bool): When True, do not provision DNS for this network.
+            driver (str): Which network driver to use when creating network.
+            enable_ipv6 (bool): Enable IPv6 on the network.
+            ingress (bool): Ignored, always False.
+            internal (bool): Restrict external access to the network.
+            ipam (IPAMConfig): Optional custom IP scheme for the network.
+            labels (Dict[str, str]):  Map of labels to set on the network.
+            macvlan (str):
```

Comment:
any mini-description here?

Reply:
I'll get one from @baude RSN and update in later PR